external/curl/Kconfig

```diff
@@ -123,7 +123,7 @@ if SSL_SUPPORT
 config MBEDTLS
 	bool "Mbed TLS"
 	default y
-	depends on !NET_SECURITY_TLS
```

Comment:
Does this enable mbedtls ported into TizenRT? NET_SECURITY_TLS seems a flag to select which mbedtls is built.